src/sentry/tsdb/redis.py

```diff
@@ -485,16 +485,17 @@ def get_frequency_series(self, model, items, start, end=None, rollup=None):
 
         # Freeze ordering of the members (we'll need these later.)
         for key, members in items.items():
-            items[key] = tuple(members)
+            items[key] = list(members)
```

Comment:
This ensures that [the `zip` call on line 507](https://github.com/getsentry/sentry/pull/5171/files/bff71630f62be95965501fcc397afbca408fef4d#diff-b85fd211c05e43b7872d7045b417b454R507) is called with the items in the same order in which they were [provided to Redis on line 498](https://github.com/getsentry/sentry/pull/5171/files/bff71630f62be95965501fcc397afbca408fef4d#diff-b85fd211c05e43b7872d7045b417b454R498). If that collection (`members` here) was cast to a list twice, the results would *probably* be the same, but this should be safer.